oshdb-util/src/main/java/org/heigit/bigspatialdata/oshdb/util/geometry/OSHDBGeometryBuilder.java

```diff
@@ -279,24 +279,31 @@ private static Geometry getMultiPolygonGeometry(
       T entity, OSHDBTimestamp timestamp, TagInterpreter areaDecider, P clipPoly) {
     Geometry geom = OSHDBGeometryBuilder.getGeometry(entity, timestamp, areaDecider);
     return Geo.clip(geom, clipPoly);
+  } 
+  
+  /**
+   * Create the Geometry of an OSHDBBoundingBox. Will always contain 5 Coordinates, even for
+   * point-like bbx -> point-like polygons for bbx of a Point.
```

Comment:
I think we could write that the result is always a rectangular _polygon_ with exactly 4 vertices. (5 coordinate geometries could be many things :sweat_smile: )

Reply:
well... the check for geometry.isRectangular() does not always return true (eg. if the bbox has the form of a line). But one of the first checks isRect.. does is _does it have exactly 5 coordinates) so I used this more generic description

Reply:
clarified in javadoc